### PR TITLE
feat: add support of clearMeasures (modified)

### PR DESCRIPTION
**Pull request by nolanlawson**
Modification of #10. Opening a separate PR so that SauceLabs will actually run the browser tests...

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -26,8 +26,9 @@ function insertSorted (arr, item) {
 let mark
 let stop
 let getEntries
+let clear
 
-if (perf && perf.mark && perf.measure) {
+if (perf && perf.mark) {
```
Comment (nolanlawson) [inline src/index.js]: pretty sure we only need to check one API here

**Comment by nolanlawson** [comment]
Passing in all browsers, I'm happy to merge now. @oaleynik ?

**Comment by oaleynik** [comment]
@nolanlawson perfect 👌🏻

**Comment by nolanlawson** [comment]
Awesome, thanks a lot for the PR!

**Comment by oaleynik** [comment]
🤘🏻

**Pull request merged**